pkg/app/client.go

```diff
@@ -0,0 +1,12 @@
+package app
+
+import "os/exec"
+
+func GetCOEVersion() ([]byte, error) {
+	return runCommand("oc", "version")
+}
+
+func runCommand(cmd string, args ...string) ([]byte, error) {
+	output, err := exec.Command(cmd, args...).CombinedOutput()
+	return output, err
+}
```

Comment:
I'll allow it, we should probably put all the runCommands in a common util package somewhere for both broker and apb packages to use.